project/categories/factory.py

```diff
@@ -0,0 +1,15 @@
+import factory
+
+from .models import Category
+
+
+class CategoryFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = Category
+
+    name = factory.Faker("name")
+
+
+categories = CategoryFactory.create_batch(10)
+for category in categories:
+    print(category.name)
```

Comment:
I don't think we want to add these lines, since we will want to generate the fake content via a manual command.
```suggestion
```